src/Module/Settings/Display.php

```diff
@@ -224,10 +240,21 @@ protected function content(array $request = []): string
 			BBCode::PREVIEW_LARGE    => $this->t('Large Image'),
 		];
 
-		$network_timelines = $this->pConfig->get($uid, 'system', 'network_timelines', array_keys($this->getAvailableTimelines($uid, true)));
+		$bookmarked_timelines = $this->pConfig->get($uid, 'system', 'network_timelines', array_keys($this->getAvailableTimelines($uid, true)));
```

Comment:
If this was an object inheriting from `BaseCollection`, you could write this instead:
```suggestion
		$bookmarked_timelines = $this->pConfig->get($uid, 'system', 'network_timelines', $this->getAvailableTimelines($uid, true)->column('code');
```

Reply:
Done